tests/test_expand.py

```diff
@@ -203,6 +211,22 @@ def test_expand_full(tmp_path: pathlib.Path, cfg_name: str, template: str):
                 assert output == expected, failure_info
 
 
+def maybe_skip(template_file: pathlib.Path) -> None:
+    """
+    Skip this test if we know it can't be done.
+
+    For example, tests that reference files from the CDS system can't be run
+    on CI. They need to have access to NFS.
+    """
+    if on_cds_nfs:
+        return
+    with open(template_file, "r") as fd:
+        if "$$INCLUDE" in fd.read():
```

Comment:
Counterintuitively, there are a number of builds that run just fine on CI despite having includes.
Maybe it's worth taking a moment to understand this behavior and only skip the ones that e.g. include absolute paths.

Reply:
All of the builds that pass on CI despite having an `$$INCLUDE` pass because the include is part of an if block that evaluates to `False` and is skipped

Reply:
I've decided to only do the skip on include check if the line count is different (e.g. if the generated file is missing a 10-line include file, the resulting expanded template would be 10-lines shorter).

Therefore, IOCs with unused `$$INCLUDE` macros (or ones that work) won't be unnecessarily skipped.

Reply:
Now we skip 8 instead of like 40